deputy/Makefile

```diff
@@ -13,6 +13,9 @@ egg local:
 	rm -rf ./dist
 	python2 setup.py bdist_egg
 	python3 setup.py bdist_egg
+	make _symlinks
```

Comment:
I'd consider revising it into dependent rules instead of such direct call. There's always a danger of creating loops.
How about creating a new rule (.e.g 'dist' or '_dist') and make sure that 'egg' & 'local' are depended on it. (Same for rest of the code)

Reply:
fixed